definitions/checks/backup/incremental_parent_type.rb

```diff
@@ -0,0 +1,36 @@
+module Checks::Backup
+  class IncrementalParentType < ForemanMaintain::Check
+    metadata do
+      description 'Check if the incremental backup has the right type'
+      tags :backup
+      param :incremental_dir, 'Path to existing backup directory'
+      param :online_backup, 'Select for online backup', :flag => true, :default => false
```

Comment:
Is the supposition that if this is not set, it defaults to offline backup?

Reply:
Yes